src/nunit.analyzers/ConstraintUsage/BaseConditionConstraintCodeFix.cs

```diff
@@ -126,13 +127,26 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
                 ? assertNode.ArgumentList.Arguments.Skip(2)
                 : assertNode.ArgumentList.Arguments.Skip(1);
 
+            var actualArgument = SyntaxFactory.Argument(actual);
+            var actualArgumentWithRightTrivia = conditionNode is not null
+                ? actualArgument.WithLeadingTrivia(conditionNode.GetLeadingTrivia()) // ignore the trailing trivia, as there is a following argument
+                : actualArgument;
+
+            var lastOriginalArgument = assertNode.ArgumentList.Arguments.Last();
+            var constraintArgument = SyntaxFactory.Argument(constraintExpression).WithTriviaFrom(lastOriginalArgument);
+            var constraintArgumentWithRightTrivia = remainingArguments.Any()
+                ? constraintArgument.WithoutTrailingTrivia() // remove the trailing trivia, as there is a following argument
+                : constraintArgument;
+
             var newArguments = new[]
             {
-                SyntaxFactory.Argument(actual),
-                SyntaxFactory.Argument(constraintExpression)
+                actualArgumentWithRightTrivia,
+                constraintArgumentWithRightTrivia
             }.Union(remainingArguments);
 
-            var newArgumentsList = assertNode.ArgumentList.WithArguments(newArguments);
+            var newArgumentsList = assertNode.ArgumentList
+                .WithArguments(newArguments)
+                .WithAdditionalAnnotations(Formatter.Annotation);
```

Comment:
The latter should be added to the `WithArguments` extension method as that method destroys any existing Trivia if it is not a new-line and therefore requires formatting.